_data/simple-icons.json

```
@@ -7082,6 +7082,17 @@
             "source": "https://www.progress.com/",
             "guidelines": "https://www.progress.com/legal/trademarks/trademarks-use-policy"
         },
+        {
+            "title": "Progressive Web Application",
+            "aliases": {
+                "aka": [
+                    "pwa"
+                ]
+            },
+            "hex": "5A0FC8",
+            "source": "https://github.com/webmaxru/progressive-web-apps-logo/blob/77744cd5c0a4d484bb3d082c6ac458c44202da03/pwalogo-white.svg",
+            "guidelines": "https://github.com/webmaxru/progressive-web-apps-logo#description"
+        },
```

Concrete requests:

- For the guidelines, I would omit the `#description` anchor. While I see why you included it, it doesn't seem very stable to me and there's not a lot of content before it. Instead, we can skip the files and folders by using the `#readme` anchor.
- We're not very consistent with this in the project, but since we're making change here anyway I propose we move the `"aliases"` key to the end.

```suggestion
            "title": "Progressive Web Application",
            "hex": "5A0FC8",
            "source": "https://github.com/webmaxru/progressive-web-apps-logo/blob/77744cd5c0a4d484bb3d082c6ac458c44202da03/pwalogo-white.svg",
            "guidelines": "https://github.com/webmaxru/progressive-web-apps-logo#readme",
            "aliases": {
                "aka": [
                    "pwa"
                ]
            }
        },
```